src/ValidatedNumerics.jl

```diff
@@ -26,25 +22,31 @@ import Base:
     ⊆, eps,
     floor, ceil, trunc, sign, round,
     expm1, log1p,
-    isfinite, isnan
+    isfinite, isnan,
+    precision
+
+if VERSION >= v"0.5.0-dev+1182"
+    import Base: rounding, setrounding, setprecision
+else
+    import Compat: rounding, setrounding, setprecision
+end
+
 
 export
     Interval,
     @interval, @biginterval, @floatinterval, @make_interval,
-    get_interval_rounding, set_interval_rounding,
     diam, radius, mid, mag, mig, hull, isinside,
     emptyinterval, ∅, ∞, isempty, interior, isdisjoint, ⪽,
     precedes, strictprecedes, ≺,
     entireinterval, isentire, nai, isnai, isthin, iscommon,
     widen, infimum, supremum,
-    set_interval_precision, get_interval_precision,
-    with_interval_precision,
     parameters, eps, dist, roughly,
     pi_interval,
     midpoint_radius, interval_from_midpoint_radius,
     RoundTiesToEven, RoundTiesToAway,
     cancelminus, cancelplus, isunbounded,
-    .., @I_str
+    .., @I_str,
+    precision, setprecision, setrounding, rounding
```

Comment:
I believe we have to reexport them if they were imported from `Compat` (rather than `Base`), so that the user doesn't have to do `using Compat` in order to use them.